doc/api/deprecations.md

```diff
@@ -1038,14 +1051,18 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Runtime
 
-The [`util.isDate()`][] API is deprecated.
+The [`util.isDate()`][] API is deprecated. Please use
+`arg instanceof Date` instead.
```

Comment:
`util.isDate` isn't equivalent to `instanceof`, though, especially across realms like worker threads. Other than npmjs.com/is-date, i don't think there _is_ an equivalent in userland.

Same applies to any `instanceof` recommendation, since instanceof doesn't work across realms, and is brittle and user-forgeable.

Reply:
oof, ok - certainly if they're not checking internal slots, then the recommendation should just be whatever it's currently doing, thanks for clarifying.